mmv1/products/redis/Cluster.yaml

```
@@ -250,6 +251,36 @@ properties:
     description: |
       System assigned, unique identifier for the cluster.
     output: true
+  - name: 'automatedBackupConfig'
+    type: NestedObject
+    description: "The automated backup config for a instance."
+    custom_flatten: 'templates/terraform/custom_flatten/redis_cluster_automated_backup_config.go.tmpl'
+    custom_expand: 'templates/terraform/custom_expand/redis_cluster_automated_backup_config.go.tmpl'
+    properties:
+      - name: 'fixedFrequencySchedule'
+        type: NestedObject
+        description: "Optional. Trigger automated backups at a fixed frequency."
```

```suggestion
        required: true
        description: "Trigger automated backups at a fixed frequency."
```

> "Error creating Cluster: googleapi: Error 400: invalid argument: invalid automated backup config: start time must be specified"

when I omit this locally. We can remove this if other options are added later.

---

Fixed, this should be required thanks for the catch